src/actions/updateIPAddrAction.js

```diff
@@ -0,0 +1,4 @@
+export const updateIPAddr = (ipAddr) => ({
```

Comment:
we can put both "updateIPAddr" and "updateUUID" into one action named userAction